docs/UserGuide.md

```diff
@@ -330,40 +334,54 @@ Format: `clear`
 
 #### Containing an open-ended question : `test INDEX ans/ANSWER` <a name="testing-oeq"></a>
 
-<div markdown="block" class="alert alert-info">
-:bulb: Answer is <b>case insensitive</b>.
-</div>
-<div markdown="block" class="alert alert-danger">
-The index <b>must be a positive integer</b> 1, 2, 3, ...
-</div>
-
 You can test yourself with a flashcard containing open-ended question by specifying an answer.
 
-1. Use the `list` command to first list all the flashcards. You can also use the `find` command to filter for a list of flashcards.
+1. Use the `list` command to first list all the flashcards. You can also use the `find` command to filter for a list of flashcards.s
+
     ![TestStep1](./images/TestStep1.png)
+    
 2. Using the indices of the displayed list, enter the `test` command followed by the index of the flashcard you want to test and what you think the answer to the question is. For example, if you want to test the second flashcard in the displayed list with the answer `a computer organization module`, you can enter `test 2 ans/a computer organization module`.
+
+    <div markdown="block" class="alert alert-info">
+    :bulb: Answer is <b>case insensitive</b>.
+    </div>
+    <div markdown="block" class="alert alert-danger">
+    The index <b>must be a positive integer</b> 1, 2, 3, ...
```

Comment:
I think can add an `:exclamation:`❗ emoji for this alert class. Same for the rest of the `alert-danger` classes